webgpu.h

```diff
@@ -113,6 +113,8 @@ typedef enum WGPUBindingType {
     WGPUBindingType_Sampler = 0x00000003,
     WGPUBindingType_SampledTexture = 0x00000004,
     WGPUBindingType_StorageTexture = 0x00000005,
```

Comment:
Would be great at least to avoid mixing the stuff we add in native with the base stuff from the Web, i.e. move it outside like
```
WGPUBindingType_StorageTexture = 0x000000010;
```